packages/automated_actions/automated_actions/config.py

```diff
@@ -9,7 +9,7 @@ class Settings(BaseSettings):
     debug: bool = False
     url: str = "http://localhost:8080"
     root_path: str = ""
-    environment: str
+    environment: str = "development"
```

Comment:
Can we live without a default here? Just to make sure it is set and we protect a bit our dynamo tables?

Reply:
Ok. Reverted in 1cb18fd